parlai/agents/seq2seq/seq2seq.py

```diff
@@ -38,17 +38,17 @@ class Seq2seqAgent(TorchAgent):
       `(Luong et al. 2015) <arxiv.org/abs/1508.04025>`_
     """
 
-    # version 1 split from version 0 on Aug 24, 2018
+    # version 1 split from version 0 on Aug 29, 2018
     # to use version 0, use --model legacy:seq2seq:0
     # legacy agent code is located in parlai/legacy_agents
     VERSION = 1
 
-    @staticmethod
-    def add_cmdline_args(argparser):
+    @classmethod
+    def add_cmdline_args(cls, argparser):
         """Add command-line arguments specifically for this agent."""
         agent = argparser.add_argument_group('Seq2Seq Arguments')
         agent.add_argument('--init-model', type=str, default=None,
-                           help='load dict/features/weights/opts from this file')
+                           help='load dict/model/opts from this path')
```

Comment:
I really like the way when we have automatic default in the help with `%(default)s`, prolly we can fix that in seq2seq now

Reply:
yes let's do separate diff for those, I also want to reformat the whitespace on all the args